edge/pkg/edged/edged_status.go

```diff
@@ -51,8 +51,8 @@ import (
 	"github.com/kubeedge/kubeedge/pkg/util"
 )
 
-//GPUInfoQueryTool sets information monitoring tool location for GPU
-var GPUInfoQueryTool = "/var/IEF/nvidia/bin/nvidia-smi"
+//IEFGPUInfoQueryTool sets information monitoring tool location for GPU
+var IEFGPUInfoQueryTool = "/var/IEF/nvidia/bin/nvidia-smi"
```

Comment:
We should remove ief related codes here if possible.

Reply:
okey

Reply:
@fisherxu GPU resource computing is expected to be solved as soon as possible